src/components/Heading/__stories__/Heading.stories.mdx

```diff
@@ -195,10 +192,9 @@ We support two kinds of ellipsis: single-line ellipsis with a tooltip displayed
         Heading with ellipsis and tooltip when hovering
       </Heading>
       <div>
-        <Heading type={Heading.types.H2} ellipsis withoutTooltip maxLines={2}>
-          Heading with two lines overflow heading without tooltip heading with two lines overflow heading without
-          tooltip heading with two lines overflow heading without tooltip heading with two lines overflow heading
-          without tooltip
+        <Heading type={Heading.types.H2} maxLines={2}>
```

Comment:
I don't understand why we set it to `withoutTooltip` before, it has no point of not showing the tooltip as it's the default when there's an overflow